### DISCUSSION TITLE
MRF searchability

### DISCUSSION THREAD

**Discussion by SBernstetter** [CMSgov/hospital-price-transparency#81] [Q&A] [answered]
May we zip or otherwise compress our MRFs? Several of ours contain >3M lines and cannot be hosted unzipped. I found a similar question on https://github.com/CMSgov/price-transparency-guide/discussions/59#discussioncomment-642881 regarding transparency in coverage that permits compression for insurers, but not clear whether this also applies to hospitals. Thanks.

**Comment by thomprl** [comment]
I have the same question.  Is it ok to zip the file as it's rather large.

**Reply by RidenourK** [reply, marked as answer]
@SBernstetter and @thomprl. Thank you for your questions.
 
The Hospital Price Transparency Rule does not restrict hospitals from zipping, or compressing for storage, the machine-readable file. We recommend if a hospital chooses to zip or compress a file that the hospital use the CMS-specified naming convention for the file (Refer to § 180.50(d)(5)).
